pokemongo_bot/__init__.py

```diff
@@ -581,7 +581,7 @@ def get_player_info(self):
                                                 ' | Pokestops Visited: {poke_stop_visits}'.format(**playerdata), 'cyan')
 
     def has_space_for_loot(self):
-        number_of_things_gained_by_stop = 5
+        number_of_things_gained_by_stop = 8
```

Comment:
What happens when you have 5 spots left but it gives you 8 items? Don't you just fill up to the max? I don't think that would be too big of a deal